frontend/packages/ux-editor/src/testing/layoutMock.ts

```diff
@@ -13,7 +13,7 @@ export const layout1NameMock = 'Side1';
 export const layout2NameMock = 'Side2';
 export const layoutSet1NameMock = 'test-layout-set';
 export const layoutSet2NameMock = 'test-layout-set-2';
-
+export const dataModelNameMock = 'test-data-model';
```

Comment:
I know that this will be a bit out of scope for your PR, but I think we should place this, and the other consts that are connected to the layoutSet to a new mock file names `layoutSetMock.ts`.

That would include this one, `layoutSet1NameMock`, `layoutSet2NameMock` and `layoutSetsMock`

Reply:
Good suggestion! The new const was a little out of place in this file. Will fix! 😊

Reply:
For both ux-editor and ux-editor-v3